### ISSUE TITLE
feedback på statistik feature

### ISSUE THREAD

**Issue by Siggik** [ronan-mch/det_sker#357]
Umiddelbart ser det super fedt ud! Dog lidt svært at gennemskue det helt på staging. Kan man f.eks. se views på de enkelte events?

**Comment by ronan-mch** [comment]
nej, p.t. er det kun overordnet. jeg har overvejet at man kunne lave en
tæller på de enkelte events hvis der er interesse i det. Skal jeg deploye
til produktion?

**Comment by Siggik** [comment]
Ja gør endelig det. Så kan vi prøve det i praksis og se på om vi har flere ønsker.

**Comment by ronan-mch** [comment]
ok, det når jeg forhåbentlig i aften.

**Comment by tadoh** [comment]
glæder mig :D

Den 17-06-2016 kl. 10:08 skrev Ronan McHugh:
> ok, det når jeg forhåbentlig i aften.
>

**Comment by ronan-mch** [comment]
husk nu at vi ville kun kunne måle besøg fra når koden er oppe og ikke over
hele projektets liv tid.

**Comment by ronan-mch** [comment]
ok, den er oppe nu.

**Comment by dotnetCarpenter** [comment]
@ronan-mch må jeg kigge med?

**Comment by ronan-mch** [comment]
Hej jon, du skal være admin. Har du en Duk Op bruger?

Den søn. 19. jun. 2016 03.25 skrev Jon Ege Ronnenberg <
notifications@github.com>:

**Comment by dotnetCarpenter** [comment]
det har jeg nu :) dotnetCarpenter

**Comment by dotnetCarpenter** [comment]
hov der er ingen domæne navn i jeres confirmation email.

Jeg får https://user/confirmation?confirmation_token?

**Comment by ronan-mch** [comment]
Fuck, tak for advarslen. Jeg retter.

Den søn. 19. jun. 2016 13.49 skrev Jon Ege Ronnenberg <
notifications@github.com>:

**Comment by ronan-mch** [comment]
Hej Jon, jeg prøvede lige med en test konto og linket kom fint igennem. Den
er også konfigureret ordentlig på serveren så vidt som jeg kan se, så jeg
aner ikke hvad fejlen kunne være... men du er nu admin så får du lov til at
kigge i alle vores hemmelige steder...

**Comment by Siggik** [comment]
Det her er optur!

**Comment by Siggik** [comment]
Ok, mere feedback på analytics. Først. Det er optur!  Det er dejligt at kunne se at folk bruger siden! Det er blevet min nye yndlings oversprings handling at tjekke analytics ud flere gange om dagen, den er blevet brugt 44 gange allerede! :)

F.eks. er det sjovt at se at der er mange der bruger lokationsiderne. Måske en grund til at rykke #259 op på to do listen? Selvom de statiske sider dog kun har 37 views. Det kunne være spændenden at se hvilke af dem folk ser på. 

Et ønske: Kan klokkeslet/dage/uger listes kronologisk under graferne? Lige nu ser de ud til at at de står i en random rækkefølge.

**Comment by Siggik** [comment]
Og hvorfor viser den aldrig 0 views? Er der aldrig 0 views eller er det mindste den kan vise 1? Eller viser den så bare ingen ting?

**Comment by Siggik** [comment]
Jeg tror at analytics ur er indstillet til en anden tidszone. Kan det omstilles til dansk tid? Så kan time funktionen blive lidt nemmere at gennemskue. F.eks. hvornår på dagen, det er folk kigger på siden etc.
